docs/configuration_guide.md

```diff
@@ -180,6 +180,20 @@ BASIC_MODEL:
   api_key: $AZURE_OPENAI_API_KEY
 ```
 
+### How to configure context length for different models
+
+Different models have different context length limitations. DeerFlow provides a method to control the context length between different models. You can configure the context length between different models in the `conf.yaml` file. For example:
+```yaml
+BASIC_MODEL:
+  base_url: https://ark.cn-beijing.volces.com/api/v3
+  model: "doubao-1-5-pro-32k-250115"
+  api_key: ""
+  token_limit: 128000
+```
+This means that the context length limit using this model is 128k. 
+
+The default value is `128000` if not specified.
```

Comment:
@Fancy-hjyp 
We need to update the document if the token_limit is not set. 
Current there is no default value of token_limit, and the context management doesn't work if the token_limit is not set.